AutoRest/Generators/NodeJS/NodeJS.Tests/AcceptanceTests/acceptanceTests.js

```diff
@@ -599,6 +601,44 @@ describe('nodejs', function () {
       });
     });
 
+    describe('Duration Client', function () {
+      var testClient = new durationClient(baseUri, clientOptions);
+      it('should properly handle null value for Duration', function (done) {
+        testClient.duration.getNull(function (error, result) {
+          should.not.exist(result);
+          should.not.exist(error);
+          done();
+        });
+      });
+
+      it('should properly handle invalid value for Duration', function (done) {
+        testClient.duration.getInvalid(function (error, result) {
+          //TODO: There should be an error here, except for some reason moment.js allows non-ISO strings and will just construct a duration of length 0
```

Comment:
I believe this is just the way the Moment library works (i.e. the behavior observed is expected).  The comment was more aimed at pointing out my original assumption was wrong... I think the comment I added is unclear, I'll remove it.